server/utils/premises/occupancy.ts

```diff
@@ -98,3 +110,101 @@ export const daySummaryRows = (daySummary: Cas1PremisesDaySummary) => {
     ],
   }
 }
+
+const itemListHtml = (items: Array<string>): { html: string } =>
+  htmlValue(`<ul class="govuk-list govuk-list">
+    ${items.map((item: string) => `<li>${item}</li>`).join('')}
+  </ul>
+`)
+
+type PlacementColumnField = Cas1SpaceBookingDaySummarySortField | 'spaceType'
+type OutOfServiceBedColumnField = keyof Cas1OutOfServiceBedSummary
+
+export type ColumnDefinition = {
+  title: string
+  fieldName: PlacementColumnField | OutOfServiceBedColumnField
+  sortable: boolean
+}
```

Comment:
This could use a type parameter maybe:

```suggestion
export type ColumnDefinition<T> = {
  title: string
  fieldName: T
  sortable: boolean
}
```

then use as: 

```
export const placementColumnMap: Array<ColumnDefinition<PlacementColumnField>> = []
```

Reply:
Yup, that works